engine/access/rpc/backend/script_executor.go

```diff
@@ -119,18 +128,30 @@ func (s *ScriptExecutor) GetAccountBalance(ctx context.Context, address flow.Add
 // - Script execution related errors
 // - storage.ErrHeightNotIndexed if the data for the block height is not available
 func (s *ScriptExecutor) GetAccountKeys(ctx context.Context, address flow.Address, height uint64) ([]flow.AccountPublicKey, error) {
-	if err := s.checkDataAvailable(height); err != nil {
+	if err := s.checkHeight(height); err != nil {
 		return nil, err
 	}
 
 	return s.scriptExecutor.GetAccountKeys(ctx, address, height)
 }
 
-func (s *ScriptExecutor) checkDataAvailable(height uint64) error {
+func (s *ScriptExecutor) checkHeight(height uint64) error {
 	if !s.initialized.Load() {
 		return fmt.Errorf("%w: script executor not initialized", storage.ErrHeightNotIndexed)
 	}
 
+	// Version control feature could be disabled. In such a case, ignore related functionality.
+	if s.versionControl != nil {
```

Comment:
let's do this check after all of the existing checks. The height range messages will be more relevant if they submit an out of range height.

Reply:
can you also update the error on line 168 to use `ErrIncompatibleNodeVersion`